pipelines/internal/commands/query/query.go

```diff
@@ -37,13 +38,28 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 	if *filter != "" {
 		call = call.Filter(*filter)
 	}
-	resp, err := call.Do()
-	if err != nil {
-		return err
+
+	if *limit == 0 {
+		return nil
 	}
 
-	for _, operation := range resp.Operations {
-		fmt.Println(operation.Name)
+	pageToken := ""
```

Comment:
var pageToken string
var listed uint

Though I think 'count' is better than 'listed'